packages/core/src/k.ts

```diff
@@ -1,43 +1,14 @@
 import { StyledComponentProps } from "./styled";
 import { StyledProps, StyledKeyType, PseudoProps } from "@kuma-ui/system";
 
-const htmlTags: Array<Partial<keyof JSX.IntrinsicElements>> = [
```

Comment:
Or, Maybe `keyof JSX.IntrinsicElements` is prefer instead of `HTMLElementTagNameMap`...?